pkg/virt-api/rest/subresource_test.go

```diff
@@ -77,31 +81,35 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 		)
 	})
 
+	expectHandlerPod := func() {
+		pod := &k8sv1.Pod{}
+		pod.Labels = map[string]string{}
+		pod.Labels[v1.AppLabel] = "virt-handler"
+		pod.ObjectMeta.Name = "madeup-name"
+
+		pod.Spec.NodeName = "mynode"
+		pod.Status.Phase = k8sv1.PodRunning
+		pod.Status.PodIP = "10.35.1.1"
```

Comment:
Where did this IP come from?

Reply:
it's just a random IP for this fake handler pod. I did not change anything here, see deleted lines below, I just made it reusable.